### PR TITLE
Public demo summary 092022

### PR DESCRIPTION
**Pull request by kpodemski**
Summary of the demo #9 of 2022

### CODE REVIEW

File: content/news/public-oss-demo/2022-12-15-public-demo-9-2022.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Also, you can take a look at the slides presented during the demo [here](https://docs.google.com/presentation/d/1SL8KE9TySjhAWJNWzjF_QJSDa4PdL60905oKSrXy_mE/edit?usp=sharing).
+
+All project members are available for feedback about the project and the software. We also invite everyone to join us on [Slack](https://www.prestashop-project.org/slack/).
+
+## Next session in January
+
+Save the date: the next demo will be streamed live on [**Wednesday, January 11th, 2022, at 4 PM CET**](https://www.youtube.com/watch?v=YDS51o48W1I). See you then!
+
+I would like to wish everyone Happy New Year, and for those who are celebrating, Merry Christmas. All the best!
```
Comment (coffrep) [inline content/news/public-oss-demo/2022-12-15-public-demo-9-2022.md]:
```suggestion
I would like to wish everyone a Happy New Year, and for those who are celebrating, Happy Holidays. All the best!
```

File: content/news/public-oss-demo/2022-12-15-public-demo-9-2022.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+authors: [ kpodemski ]
+icon: icon-video-camera
+tags: [contribute, event, community]
+---
+
+The project's public demo was held for the ninth time this year and [streamed live](https://www.youtube.com/watch?v=yN4k2-3onCA) on December 14th, 2022.
+
+The session's agenda was as follows:
+
+- [00:00](https://www.youtube.com/watch?v=yN4k2-3onCA) Beginning of the session: the introduction
```
Comment (coffrep) [inline content/news/public-oss-demo/2022-12-15-public-demo-9-2022.md]:
```suggestion
- [00:00](https://www.youtube.com/watch?v=yN4k2-3onCA) Beginning of the session: introduction
```

**Review by coffrep — APPROVED** [review]
Hi - I've added two suggestions, trying to be more "inclusive" with the happy holidays, let me know if this does or doesn't make sense. Approving with feedback

**Comment by kpodemski** [comment]
Good idea, thanks @coffrep !

**Pull request merged**